app/forms/schools/add_participant_form.rb

```diff
@@ -358,5 +358,9 @@ def appropriate_body_selected
     rescue ActiveRecord::RecordNotFound
       nil
     end
+
+    def existing_participant_profile_same_school?
+      existing_participant_profile&.current_induction_record&.school == school_cohort.school
```

Comment:
I wonder if it'd be better to use the latest induction record no matter its induction_status:
`existing_participant_profile&.induction_records&.latest&.school == school_cohort.school`